backend/routes/productRoutes.js

```diff
@@ -1,10 +1,51 @@
 import express from 'express';
 const router = express.Router();
-import { Product } from "../models/productModel.js";
-//GET /products
+import { Product } from '../models/productModel.js';
+
+/**
+ * @swagger
+ * /products:
+ *   get:
+ *     summary: Retrieve all products from the database
+ *     description: Retrieve all products from the database. Can be used to populate our product page.
+ *     responses:
+ *      200:
+ *        description: a list of products
+ *        content:
+ *           application/json:
+ *             schema:
+ *               type: array
+ *               items: 
+ *                $ref: '#/components/schemas/Product'
+ */
+
+/**
+ * @swagger
```

Comment:
we might be able to document the model in the model file?

Reply:
can you give an example of how to do that?